.github/workflows/testing.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       - name: Install clang++
         run: sudo apt-get install clang-${{ env.clangppDefaultValue }}++
       - name: Run tests
-        env:
-          CXX: clang++-${{ env.clangppDefaultValue }}
-        run: python runTests.py -j 2 src/test/unit
+        run: |
+          echo "CXX=clang-${{ env.clangppDefaultValue }}" >> make/local
```

Comment:
isn't better to specify

```bash
echo "CXX=clang++-${{ env.clangppDefaultValue }}" >> make/local
```

than

```bash
echo "CXX=clang-${{ env.clangppDefaultValue }}" >> make/local
```

Reply:
You're right, most probably a typo. Thanks!